fetch_submissions.py

```diff
@@ -69,7 +69,10 @@ def login(username, password):
       'next': '/'
   }
 
-  CLIENT.post(LOGIN_URL, data=payload, headers=dict(Referer=LOGIN_URL))
+  response = CLIENT.post(LOGIN_URL, data=payload, headers=dict(Referer=LOGIN_URL))
+  if response.url == "https://leetcode.com/accounts/login/":
+    print("The e-mail address and/or password you specified are not correct.", file=sys.stderr)
+    sys.exit(1)
```

Comment:
I don't think the script should exit. Instead, we should ask the user to key in the password again. Maybe in `__main__`, you could have something like:
```python
did_login = False

while not did_login:
  password = getpass.getpass()
  did_login = login(username, password)

  if not did_login:
    print("Login Failed! Please enter your password again.")
```

Reply:
I think it's better in that case for the script to prompt the user to enter his username 
(in place of passing it as a script argument) and password together if any of them are entered incorrectly. I also think that we should set a threshold for unsuccessful login attempts (my suggestion is a maximum of 5).

Reply:
Also, the error message I typed is the exact one shown on unsuccessful login on LeetCode. I think it's a good error message because it's general and doesn't indicate whether it's the username entered wrongly or the password.

Reply:
I agree with the first part. Maybe we should prompt for both username and password and have a threshold.

As for the second one, sure but the error message says "e-mail address".

Reply:
The error message is `The e-mail address and/or password you specified are not correct.`

Reply:
Why does it matter if we allow login with username and not email? I don't see much point in doing more work just match the error message from Leetcode

Reply:
I have changed it to username. I realised that LeetCode API allows username scope requests, so the user can't login with his/her email address anyways.